components/CheckDocumentation/connections/BlockingQuery.tsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+
+import SQL from "../../SQL";
+
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const BlockingQueryTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  return (
+    <>
+      <p>
+        Detects queries currently blocking equal or more than the specified
+        threshold of <code>{config.settings["blocked_count"]}</code> queries.
+        When any blocked query is running for longer than the specified
+        threshold of <code>{config.settings["warning_blocked_age_secs"]}</code>{" "}
+        seconds, creates an issue with severity "warning". Escalates to
+        "critical" if any blocked query continues to run longer than{" "}
+        <code>{config.settings["critical_blocked_age_secs"]}</code> seconds.
+        Resolves automatically once these queries complete and release their
```

Comment:
I suppose we should say
```suggestion
        Resolves automatically once these queries complete or abort and release their
```
for completeness.